tools/abel/validators/README.md

```diff
@@ -0,0 +1,210 @@
+# ABEL Spreadsheet
+
+### Overview
+The ABEL Spreadsheet serves as a frontend for the ABEL application. The
+spreadsheet is the user-friendly interface that allows users to make changes to
+a Building Configuration file.
+
+## Google Sheets template
+Since the format of an ABEL spreadsheet is well-defined, it is suggested that
+you use the following [Google Sheets Template](https://docs.google.com/spreadsheets/d/1tcLjFnHiXUT-xh5C1hRKiUVaUH_CzgSI8zFQ_B8q7vs/copy#gid=980240783) as either your input or output spreadsheet.
```

Comment:
what do you mean here?
use it as an output spreadsheet?
it is generated, I dont follow the statement here,